frontend/src/components/Playback/Multiplayer.scss

```diff
@@ -29,8 +25,37 @@
 
     .instruction {
         min-height: 55px;
+
         h3 {
             margin: 0;
         }
     }
-}
+
+    &.neutral-inverted {
+
+        .player-wrapper:nth-child(1) {
+
+            .aha__play-button {
+                background-color: $blue;
+                box-shadow: 0 0 0 0.2em rgba($blue, 0.5);
+            }
+
+            .banner {
+                background-color: rgba($blue, 0.75);
+            }
+        }
+
+        .player-wrapper:nth-child(2) {
+
+            .aha__play-button {
+                background-color: $yellow;
+                box-shadow: 0 0 0 0.2em rgba($yellow, 0.5);
+            }
+
+            .banner {
+                background-color: rgba($yellow, 0.75);
+            }
+        }
+
+    }
+}
```

Comment:
Could this bit be added to the `color-schemes.scss` file instead? My idea was that we'd gather all the styling related to the `style` attributes sent from the backend in that file.

Reply:
When we're talking about the styling of the basic html building blocks like headings, paragraphs, lists, links, inputs, labels and buttons, I would agree. But this Multiplayer component's styling is so specific, with that hanging banner underneath the play button I think it would be better to keep it here. I'm not saying that it's ideal, but having to switch files every time I want to change the styling of a specific component also annoys me in advance 😅